File: 1047 - Game Time with Minutes/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int ih = scan.nextInt();
        int im = scan.nextInt();
        int fh = scan.nextInt();
        int fm = scan.nextInt();
        scan.close();
        int tm = (fh - ih) * 60 + fm - im;
        if (tm < 1) {
            tm += 1440;
        }
        System.out.printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", tm / 60, tm % 60);
    }
}

File: 1047 - Game Time with Minutes/Main.cpp
#include <iostream>
using namespace std;

int main()
{
    int ih, im, fh, fm;
    cin >> ih >> im >> fh >> fm;
    int tm = (fh - ih) * 60 + fm - im;
    if (tm < 1)
    {
        tm += 1440;
    }
    cout << "O JOGO DUROU " << tm / 60 << " HORA(S) E " << tm % 60 << " MINUTO(S)" << endl;
    return 0;
}